internal/transform/scope-css.go

```diff
@@ -131,12 +142,16 @@ outer:
 						isBracket = false
 						out += strVal
 					case "{":
-						keyframeCurlyCount++
+						if isKeyframes {
+							keyframeCurlyCount++
+						}
```

Comment:
Only track this when we're inside `isKeyframes`, otherwise we might be starting at a weird number.